### PR TITLE
Test more edge cases in PFS service

### PR DESCRIPTION
**Pull request by palango**
Add some tests for the event handlers in PFS service.

### CODE REVIEW

File: tests/pathfinding/test_service.py [hunk 1]
```diff
@@ -106,17 +107,137 @@ def new_service(filename):
...
+        channel_identifier=ChannelID(1),
+        participant1=participant1,
+        participant2=participant2,
+        settle_timeout=20,
+        block_number=BlockNumber(1),
+    )
+
+    pathfinding_service_mock.handle_event(channel_event)
+    assert len(pathfinding_service_mock.token_networks) == 1
+    assert len(token_network_model.channel_id_to_addresses) == 1
```
Comment (karlb) [inline tests/pathfinding/test_service.py]: This part is repeated three times. Reusing it would make the tests a bit shorter.
